app/Lio/ServiceProviders/HashidsServiceProvider.php

```diff
@@ -5,15 +5,11 @@
 
 class HashidsServiceProvider extends ServiceProvider
 {
-    public function boot()
+    public function register() 
     {
-        $this->app->bind('Hashids\Hashids', function() {
+        $this->app->bind(['Hashids\Hashids' => 'hashids'], function() {
```

Comment:
It doesn't really matter that much because config is always bootstrapped but yeah best to keep to best practices and just pass it through the callback.